Python/Product/Uap/Templates/Projects/BackgroundService/StartupTask.py

```diff
@@ -0,0 +1 @@
+print('Hello World')
```

Comment:
Is there anything better we can start users off with? Do they need some sort of loop or some imports? If not, we'd prefer a blank file rather than a useless print statement.